underactuated.html

```diff
@@ -491,34 +491,41 @@
 <h1>Definitions</h1>
 
 <p> According to Newton, the dynamics of mechanical systems are second order ($F =
-ma$).  Their state is given by a vector of positions, $\bq$, and a vector of
-velocities, $\dot{\bq}$, and (possibly) time. The general form for a
-second-order controllable dynamical system is: $$\ddot{\bq} = {\bf
+ma$).  Their state is given by a vector of positions, $\bq$ (also known as the configuration vector),
+and a vector of velocities, $\dot{\bq}$. The general form for a (time-variant)
+second-order dynamical system with control inputs is: $$\ddot{\bq} = {\bf
 f}(\bq,\dot{\bq},\bu,t),$$ where $\bu$ is the control vector.  As we will see,
 the dynamics for many of the robots that we care about turn out to be affine in
 commanded torque, so let's consider a slightly constrained form:
+
+<div data-type="definition"><h1>Second-order control-affine system</h1>
+A second-order control-affine system is a dynamical system that can be described by
+the second-order differential equation
 \begin{equation}\ddot{\bq} = {\bf f}_1(\bq,\dot{\bq},t) + {\bf
-f}_2(\bq,\dot{\bq},t)\bu \label{eq:f1_plus_f2}.\end{equation} </p>
+f}_2(\bq,\dot{\bq},t)\bu \label{eq:f1_plus_f2}.\end{equation} </div>
+
+We can now define two useful, mutually exclusive properties of such a system at a
+given state and time.
 
-<div data-type="definition"><h1>Fully-Actuated</h1> A control system described
-by equation \ref{eq:f1_plus_f2} is fully-actuated in configuration
-$(\bq,\dot{\bq},t)$ if it is able to command an instantaneous acceleration in an
+<div data-type="definition"><h1>Fully-Actuated</h1> A dynamical system described
+by equation \ref{eq:f1_plus_f2} is fully-actuated in state
+$(\bq,\dot{\bq})$ at time $t$ if it is able to command an instantaneous acceleration in an
 arbitrary direction in $\bq$: \begin{equation} \textrm{rank}\left[{\bf f}_2
 (\bq,\dot{\bq},t)\right] = \dim\left[\bq\right].\end{equation} </div>
 
-<div data-type="definition"><h1>Underactuated</h1> A control system described by
-equation \ref{eq:f1_plus_f2} is underactuated in configuration
-$(\bq,\dot{\bq},t)$ if it is not able to command an instantaneous acceleration
+<div data-type="definition"><h1>Underactuated</h1> A dynamical system described by
+equation \ref{eq:f1_plus_f2} is underactuated in state
+$(\bq,\dot{\bq})$ at time $t$ if it is not able to command an instantaneous acceleration
 in an arbitrary direction in $\bq$: \begin{equation} \textrm{rank}\left[{\bf
 f}_2(\bq,\dot{\bq},t)\right] < \dim\left[\bq\right].
 \label{eq:underactuated_def}\end{equation} </div>
 
-<p>Notice that whether or not a control system is underactuated may depend on
-the state of the system, although for most systems (including all of the systems
+<p>Notice that whether or not a dynamical system is underactuated may depend on
+the state of the system and on time explicitly, although for most systems (including all of the systems
 in this book) underactuation is a global property of the system.</p>
```

Comment:
Should we add a note stating that in general a system is underactuated if it is underactuated for *any* state or time?